environment.mk

```diff
@@ -98,8 +98,12 @@ endif # Matches: ifdef _BSG_MANYCORE_DIR
 # Undefine the temporary variable to prevent its use
 undefine _BSG_MANYCORE_DIR
 
+# If we're in the docker container, verilator is installed in /usr/bin/verilator
```

Comment:
Is this because Verilator is installed through a package manager? It is best to use the one provided/compiled by bladerunner for parity.

Reply:
No, when building the Docker image it builds Verilator from source and installs it there.

Reply:
Do we have to install it there? Can we just leave it?

Reply:
I would rather be explicit that we're using the one in bladerunner, than install it in /usr/bin and be implicit.

Reply:
Well the point is to not have the user build Verilator themselves - they just download the image and it'll be preinstalled. We could just not install it and keep the binary in `/verilator` in the image, so we'd still be explicit.